web/src/pages/resetPassword/index.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+import { ReactComponent as IconVisibilityPassword } from '../../images/visibility_off.svg';
+import { apiRequest } from '../../services/api';
+import alertMessage from '../../components/alertMessage';
+import InputIcon from '../../components/inputIcon';
+import HomeSidebar from '../../components/homeSidebar';
+
+export default function ResetPassword() {
+  const [password, setPassword] = useState({ password: '', confirmPassword: '' });
```

Comment:
I think the name of this variable is confusing. It sounds like a simple variable but it's an object instead. Maybe `formData` or `formFields` would be better